db/changes.go

```diff
@@ -964,6 +974,7 @@ func (db *Database) SimpleMultiChangesFeed(chans base.Set, options ChangesOption
 
 		waitForChanges:
 			for {
+				db.DbStats.CBLReplicationPull().NumPullReplTotalCaughtUp.Add(1)
```

Comment:
NumPullReplTotalCaughtUp shouldn't be getting incremented below (old line number 978).